### PR TITLE
 Fix #11 Provide output path

### LINKED ISSUE #11: Add functionality to provide output path as well instead of assuming it
Output path is currently set as ```converted-text``` inside the source directory. Instead we should be taking this information from the user

### PR DESCRIPTION
**Pull request by rohithredd94**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -57,6 +56,11 @@ def main(path):
...
 	parser.add_argument("input_dir")
+	parser.add_argument('output_dir', nargs='?')
 	args = parser.parse_args()
 	path = os.path.abspath(args.input_dir)
-	main(path)
\ No newline at end of file
+	if args.output_dir:
+		directory_path = os.path.abspath(args.output_dir) + '/'
+	else:
+		directory_path = path + '/converted-text/'
```
Comment (prabhakar267) [inline main.py]: Use `os.path.join()` instead of concatenating the string
`os.path.join()` takes care of trailing slashes

File: main.py [hunk 2]
```diff
@@ -18,11 +18,10 @@ def create_directory(path):
 def check_path(path):
 	return bool(os.path.exists(path))
 
-def main(path):
+def main(path, directory_path):
```
Comment (prabhakar267) [inline main.py]: Use more informative variable names like `source_directory` and `target_directory`

File: main.py [hunk 3]
```diff
@@ -57,6 +56,11 @@ def main(path):
 if __name__ == '__main__':
 	parser = argparse.ArgumentParser()
 	parser.add_argument("input_dir")
+	parser.add_argument('output_dir', nargs='?')
```
Comment (prabhakar267) [inline main.py]: Add help strings if possible.

**Review by prabhakar267 — CHANGES_REQUESTED** [review]

File: main.py (hunk 3 above)
Comment (rohithredd94) [reply main.py]: What do you mean by add help strings? Argument parser has a flag -h which displays info about positional arguments. This is displayed on using -h flag:
usage: main.py [-h] input_dir [output_dir]

positional arguments:
  input_dir
  output_dir

optional arguments:
  -h, --help  show this help message and exit

Should I add anything else?

Comment (prabhakar267) [reply main.py]: When a user tries to see help (python main.py -h), we can have him view a custom help message for each argument.
You can have a look at the example from argparse documentation.
Example: https://docs.python.org/2/library/argparse.html#example

**Comment by rohithredd94** [comment]
@prabhakar267 I just updated the code with the requested changes. Can you please review them?

**Review by prabhakar267 — APPROVED** [review]

**Pull request merged**